apps/cowswap-frontend/src/modules/swap/containers/Row/RowFee/index.tsx

```diff
@@ -86,25 +88,27 @@ export function RowFee({
   // trades are null when there is a fee quote error e.g
   // so we can take both
   const props = useMemo(() => {
-    const displayFee = realizedFee || fee
+    const label = swapZeroFee ? 'Est. fees' : 'Fees'
+    const displayFee = realizedFee || feeAmount
     const feeCurrencySymbol = displayFee?.currency.symbol || '-'
     // TODO: delegate formatting to the view layer
-    const smartFeeFiatValue = formatFiatAmount(feeFiatValue)
-    const smartFeeTokenValue = formatTokenAmount(displayFee)
-    const feeAmountWithCurrency = `${smartFeeTokenValue} ${formatSymbol(feeCurrencySymbol)} ${
+    const feeInFiatFormatted = formatFiatAmount(feeInFiat)
+    const displayFeeFormatted = formatTokenAmount(displayFee)
+    const feeAmountWithCurrency = `${displayFeeFormatted} ${formatSymbol(feeCurrencySymbol)} ${
```

Comment:
Is this adding the fee=0 logic back? Didn't know it was removed...

Reply:
It just changes some names. Performed during the refactor of the two fee Rows